Add YAML snippet how to preserve your battery pack (#50)
syssi/esphome-jbd-bms: README.md

    direction: BOTH
```

> ## Preserving your batteries

Reviewer: Could you move this to the `Goodies` section and use a sub-headline? (`### Preserving your batteries`)

> If you wish to limit the charge of your battery without an external system like a home assistant, take inspiration from this yml :

Reviewer: ```suggestion
If you wish to limit the charge of your battery without an external system like a home assistant, take inspiration from this YAML snippet:
```

```yaml
deep_sleep:
  id: sleep_manager
  run_duration: 2min
  sleep_duration: 10min

switch:
  - platform: ble_client
    ble_client_id: client0
    name: "${name} bluetooth connection"

  - platform: jbd_bms_ble
    jbd_bms_ble_id: bms0
    charging:
      name: "${name} charging"
      id: charging
    discharging:
      name: "${name} discharging"

sensor:
  - platform: jbd_bms_ble
    jbd_bms_ble_id: bms0
    battery_strings:
      name: "${name} battery cells"
    charging_power:
      name: "${name} charging power"
    state_of_charge:

>       name: "${name} battery pourcentage"

Reviewer: ```suggestion
      name: "${name} state of charge"
```

>       id: battery_percentage

Reviewer: ```suggestion
      id: soc
```

      on_value:
        then:
          # Manage charge
          - if:
              condition:

>                 lambda: 'return id(battery_percentage).state < 80 && id(charging).state == false;'

Reviewer: ```suggestion
                lambda: 'return id(soc).state < 80 && id(charging).state == false;'
```

              then:
                - switch.turn_on: charging
          - if:
              condition:

>                 lambda: 'return id(battery_percentage).state > 80 && id(charging).state == true;'

Reviewer: ```suggestion
                lambda: 'return id(soc).state > 80 && id(charging).state == true;'
```

              then:
                - switch.turn_off: charging
          # Disable deep_sleep to stop charge at perfect timming (81% goes down to 80% after charge stop)
          - if:
              condition:

>                 lambda: 'return id(battery_percentage).state >= 75 && id(battery_percentage).state <= 81 && id(charging).state == true;'

Reviewer: ```suggestion
                lambda: 'return id(soc).state >= 75 && id(soc).state <= 81 && id(charging).state == true;'
```

May be move the last condition to the front? It's more lightweight to evaluate a bool.
